admin/aioseop_module_class.php

```diff
@@ -1785,6 +1736,16 @@ function admin_enqueue_styles( $hook_suffix ) {
 			if ( function_exists( 'is_rtl' ) && is_rtl() ) {
 				wp_enqueue_style( 'aioseop-module-style-rtl', AIOSEOP_PLUGIN_URL . 'css/modules/aioseop_module-rtl.css', array( 'aioseop-module-style' ), AIOSEOP_VERSION );
 			}
+
+			// Uses WP Scripts to load the current platform version of jQuery UI CSS .
+			$wp_scripts = wp_scripts();
+			wp_enqueue_style(
+				'aioseop-jquery-ui-css',
+				'https://ajax.googleapis.com/ajax/libs/jqueryui/' . $wp_scripts->registered['jquery-ui-core']->ver . '/themes/smoothness/jquery-ui.css',
```

Comment:
.min file should be preferred. Google does not host it but cdnjs does.

Reply:
Good catch. ...and Google does. Changed.